### ISSUE TITLE
Add feature in pytorch-lightning to implement one cycle policy for better hyperparam search

### ISSUE THREAD

**Issue by suvojit-0x55aa** [suvojit-0x55aa/3DMMasSTN-Pytorch#6]
Roadmap for feature
 - [ ] Implement per iteration callback for pytorch-lightning
 - [ ] Implement learning rate finder(one cycle policy) for pytorch-lightning. Tracked in https://github.com/williamFalcon/pytorch-lightning/issues/624
